public/angular/templates/media/settings/twitter.html

```diff
@@ -0,0 +1,53 @@
+<div>
+  <form name="settingsForm" class="form-horizontal" novalidate>
+    <div class="modal-header">
+      <button class="close" ng-click="close()" aria-hidden="true">×</button>
+      <h4 class="modal-title">
+        <div>
+          <div class="icon-left">&nbsp;Edit Twitter Settings Source</div>
+      </h4>
+      </div>
+      <div class="modal-body">
+        <alert type="info" ng-show="success">The credentials are valid. You need to save them before continuing. </alert>
+        <alert type="info" ng-show="failure">The credentials are invalid or they have expired</alert>
+        <p ng-show="settingsForm.$invalid" class=""> All fields are required.</p>
+        <p ng-show="!settingsForm.$invalid" class=""> You can now test (recommended) or save the settings.</p>
+        <div class="well bs-component">
+          <fieldset>
+            <div class="form-group" >
+              <label for="consumer_key" class="col-lg-2 control-label">Consumer Key</label>
+              <div class="col-lg-8">
+                <input type="text" ng-model="settings.consumer_key" class="form-control" id="consumer_key" name="consumer_key" placeholder="{{ settings.consumer_key }}" ng-focus required>
```

Comment:
Please be consistent with capitalization style for variables. The app uses camelCase, so please stick to that. Please fix this throughout.